### PR TITLE
Expose tsearch/ts_public.h and tsearch/ts_utils.h

### PR DESCRIPTION
**Pull request by sachaarbonel**
Related to #321

### CODE REVIEW

**Comment by eeeebbbbrrrr** [comment]
Would love to merge this if you can fix up the merge conflicts

**Comment by Hoverbear** [comment]
Thanks for this! I opened #464 since I wanted to make sure this was all clean with https://github.com/zombodb/pgx/pull/455. I'll shepherd it to merge, you can expect it in the full 0.4.0 release soon.

File: pgx-pg-sys/include/pg10.h [hunk 1]
```diff
@@ -78,6 +78,8 @@
 #include "storage/procarray.h"
 #include "tcop/tcopprot.h"
 #include "tcop/utility.h"
+#include "tsearch/ts_public.h"
```
Comment (JefryMW23) [inline pgx-pg-sys/include/pg10.h:81]: H

File: pgx-pg-sys/include/pg10.h [hunk 2]
```diff
@@ -78,6 +78,8 @@
...
+#include "tsearch/ts_utils.h"
```
Comment (JefryMW23) [inline pgx-pg-sys/include/pg10.h:82]: B

**Review by JefryMW23 — COMMENTED** [review]
G

**Pull request closed without merging**